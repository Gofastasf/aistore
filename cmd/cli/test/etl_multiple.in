// RUN k8s
# Test multiple ETL initialization in a single spec file
ais etl rm --all // IGNORE
echo -e "name: test-etl1\nruntime:\n    image: aistorage/transformer_hello_world:latest\n---\nname: test-etl2\nruntime:\n    image: aistorage/transformer_hello_world:latest" > etl-config.yaml // IGNORE
ais etl init -f etl-config.yaml --name test-etl1 2>&1 || true
ais etl init -f etl-config.yaml // IGNORE
ais etl show | grep Running | wc -l
ais etl stop -f etl-config.yaml test-etl-3 2>&1 | head -n 1
ais etl stop -f etl-config.yaml
ais etl start -f etl-config.yaml
ais etl stop test-etl1
ais etl start -f etl-config.yaml 2>&1 
ais etl rm -f etl-config.yaml test-etl-3 2>&1 | head -n 1
ais etl rm -f etl-config.yaml
ais etl show | wc -l
rm -f etl-config.yaml // IGNORE