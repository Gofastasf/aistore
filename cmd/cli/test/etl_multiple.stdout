Error: cannot use override flags [--name] with a multi-ETL file; remove these flags or split into separate files
2
Incorrect 'ais etl stop' usage: flag '--spec' cannot be used together with ETL_NAME [test-etl-3].
ETL[test-etl1] stopped
ETL[test-etl2] stopped
ETL[test-etl1] started successfully
ETL[test-etl2] started successfully
ETL[test-etl1] stopped
ETL[test-etl1] started successfully
invalid action "can't start ETL[test-etl2] during Running stage"
Incorrect 'ais etl rm' usage: flag '--spec' cannot be used together with ETL_NAME [test-etl-3].
ETL[test-etl1] successfully deleted
ETL[test-etl2] successfully deleted
0
